packages/pharos/src/components/combobox/pharos-combobox.scss

```diff
@@ -180,6 +180,19 @@
   @include selected-text;
 }
 
+.combobox__option--disabled {
+  color: var(--pharos-dropdown-menu-item-color-text-disabled);
+  background-color: var(--pharos-dropdown-menu-item-color-background-disabled);
+  cursor: default;
+
+  &:hover,
+  &[highlighted] {
```

Comment:
Not sure I understand the presence of this `[highlighted]` rule; can you elaborate?

Reply:
Ah, you are right, it can be removed. In earlier versions of this, it was possible to navigate to the disabled options and this was added to fix an issue making it no longer look disabled due to specificity issues. But that should never happen now